### PR TITLE
Add Assembler for dependencies

### PR DESCRIPTION
**Pull request by ochococo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,4 +186,54 @@ final class HAL9000ViewController: UIViewController {
...
+    func assemble() -> PodBayDoorsInteracting {
+        return PodBayDoorsInteraction()
+    }
+}
+
+protocol DiscoveryOneAssembling {
+    func assemble() -> UIViewController
+}
+
+struct DiscoveryOneAssembler: DiscoveryOneAssembling {
```
Comment (sunshinejr) [inline README.md:227]: definitely prefer the lifetime version using a `struct` than the non-lifetime `enum` version 👍

File: README.md [hunk 2]
```diff
@@ -188,4 +186,54 @@ final class HAL9000ViewController: UIViewController {
...
+### Definitions
+
+### TL;DR
+
+Its purpose is to assemble dependencies for one class or struct. 
+
+#### Assembler:
+- a struct,
+- as only one method called `assemble()`. 
+- can call ONLY ONE initializer directly,
```
Comment (sunshinejr) [inline README.md:207]: this might be a little bit too strict 😅

Comment (ochococo) [reply README.md:207]: It is 😂

**Comment by ochococo** [comment]
@sunshinejr Thank you for taking time to review 🙏

**Pull request merged**